proto/v1beta1/spdx.proto

```diff
@@ -377,33 +407,3 @@ enum RelationshipType {
   // included in the Relationship comments field
   OTHER = 43;
 }
-
-// RelationshipNote represents an SPDX Relationship
-// section:
-// https://spdx.github.io/spdx-spec/7-relationships-between-SPDX-elements/
-message RelationshipNote {
-  // The type of relationship between the source and target SPDX elements
-  RelationshipType type = 1;
-}
-
-// RelationshipOccurrence represents an SPDX Relationship
-// section:
-// https://spdx.github.io/spdx-spec/7-relationships-between-SPDX-elements/
-message RelationshipOccurrence {
```

Comment:
is this intentionally deleted?

Reply:
This message is still there. I just moved enum to the bottom of the file.

Reply:
Oh I see now, apologies, I missed it in the diff earlier.